back/app/models/custom_field.rb

```diff
@@ -80,6 +80,7 @@ class CustomField < ApplicationRecord
   scope :hidden, -> { where(hidden: true) }
   scope :support_multiple_values, -> { where(input_type: 'multiselect') }
   scope :support_single_value, -> { where.not(input_type: 'multiselect') }
+  scope :support_free_text_value, -> { where(input_type: %w[text multiline_text text_multiloc multiline_text_multiloc html_multiloc]) }
```

Comment:
I'm wondering if it could make sense to move the list to a constant, that could be reused to e.g. implement a `text?` predicate for the custom field model (which could also be included in the serializer). But probably overkill and absolutely not blocking.